src/Tests/Microsoft.NET.Pack.Tests/GivenThatWeWantToPackACrossTargetedLibrary.cs

```diff
@@ -33,11 +33,14 @@ public void It_packs_nondesktop_library_successfully_on_all_platforms()
             var outputDirectory = new DirectoryInfo(Path.Combine(libraryProjectDirectory, "bin", "Debug"));
             outputDirectory.Should().OnlyHaveFiles(new[] {
                 "NetStandardAndNetCoreApp.1.0.0.nupkg",
-                "netcoreapp1.1/NetStandardAndNetCoreApp.dll",
-                "netcoreapp1.1/NetStandardAndNetCoreApp.pdb",
-                "netcoreapp1.1/NetStandardAndNetCoreApp.runtimeconfig.json",
-                "netcoreapp1.1/NetStandardAndNetCoreApp.runtimeconfig.dev.json",
-                "netcoreapp1.1/NetStandardAndNetCoreApp.deps.json",
+                $"{ToolsetInfo.CurrentTargetFramework}/NetStandardAndNetCoreApp.dll",
+                $"{ToolsetInfo.CurrentTargetFramework}/NetStandardAndNetCoreApp.pdb",
+                $"{ToolsetInfo.CurrentTargetFramework}/NetStandardAndNetCoreApp.runtimeconfig.json",
+                $"{ToolsetInfo.CurrentTargetFramework}/NetStandardAndNetCoreApp.deps.json",
+                $"{ToolsetInfo.CurrentTargetFramework}/Newtonsoft.Json.dll",
+                $"{ToolsetInfo.CurrentTargetFramework}/NetStandardAndNetCoreApp.deps.json",
```

Comment:
Isn't there a duplicate listed here?

```suggestion
                $"{ToolsetInfo.CurrentTargetFramework}/NetStandardAndNetCoreApp.deps.json",
                $"{ToolsetInfo.CurrentTargetFramework}/Newtonsoft.Json.dll",
```